### ISSUE TITLE
Create realm management page

### ISSUE THREAD

**Issue by DonWillems** [openremote/openremote#186]
Only a superuser can see and use the Realm management page.

The realm management page is structurally the same as the user page, except for realms.
In the table the rows represent realms.
Adding a new realm requires the fields:
- Realm name
- Friendly name
- Disabled/Enabled toggle
(see [specification](https://docs.google.com/document/d/1LZOKCIng-_mssSwULkTBFgeaR5Okjo6I1gM3VEAqHvE/edit#heading=h.38g0fg5hm5jc))

The master realm can not be deleted and has no delete button.
Other realms can be deleted, a warning dialog should appear. Evaluate additional checks before deleting, e.g. The user needs to type the name of the realm in the 'Delete realm' dialog before being able to press delete.

Only the friendly name of a realm can be changed once a realm is created. Realm name becomes read-only after creation.

Verify in the Keycloak process that a realm can be deleted if it has users, or that the users need to be deleted first

**Comment by pierrekil** [comment]
- [ ] When realm is created, immediately update dropdown list of realms (now needs refresh)
- [ ] In rules LHS and RHS only assets of this realm should be shown
- [ ] Adding assets and rules as admin works
- [ ] Adding location to City asset, needs explicit setting of 'show on dashboard' (shouldn't that be default?)
- [ ] New 'write' user for new added realm logging into that new realm doesn't see assets or rules created, or assets on map; a 'read' user does see assets and rules and assets on map
- [ ] Realm management should only be in dropdown menu for the 'super admin' user. Now it's also visible for a regular 'write' user in that New realm
- [ ] Users and roles should not be available for 'read' users

**Comment by pierrekil** [comment]
- [ ] Not all new realms get updated, only the first added
- [ ] Users and roles creation is now the same option as creating realms. but we want admin users per realm to be able to create users and roles, but not realms
- [ ] When write roles are ticked, read roles should also be ticked. Do we want that to happen by default?

**Comment by pierrekil** [comment]
created separate issue for remaining item
